mesa/visualization/templates/js/InteractionHandler.js

```diff
@@ -125,8 +125,8 @@ const InteractionHandler = function (width, height, gridWidth, gridHeight, ctx)
 
       // map the event to x,y coordinates
       const position = coordinateMapper(event);
-      const yPosition = Math.floor(event.offsetY / cellHeight);
-      const xPosition = Math.floor(event.offsetX / cellWidth);
+      // const yPosition = Math.floor(event.offsetY / cellHeight);
```

Comment:
You can remove them instead. More confusing to keep them.

Reply:
removed